indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -339,6 +339,17 @@ public static Configuration injectSystemProperties(Configuration conf)
     return conf;
   }
 
+  public static Configuration injectSystemAndJobProperties(Configuration conf, HadoopDruidIndexerConfig config)
+  {
+    Configuration updatedConf = injectSystemProperties(conf);
+    HadoopIngestionSpec schema = config.getSchema();
+
+    for (final Map.Entry<String, String> entry : schema.getTuningConfig().getJobProperties().entrySet()) {
+      updatedConf.set(entry.getKey(), entry.getValue());
+    }
```

Comment:
@himanshug - I see your point regarding using existing method addJobProperties in HadoopDruidIndexerConfig, which only update job object properties.  Are your suggesting we replace the existing way of creating new Configuration object for getFileSystem(conf) with Configuration from job object? The getFileSystem(conf) is used by getPublishedSegments and runJobs methods which are static methods and job object is not easily available ( e.g call via MetadataStorageUpdaterJob.run ).